pkg/cmd/inspect/resource.go

```diff
@@ -166,27 +164,42 @@ func (o *InspectOptions) gatherNamespaceData(baseDir, namespace string) error {
 
 	log.Printf("    Collecting resources for namespace %q...\n", namespace)
 
+	// "all" maps to every resource we wish to collect for the given namespace,
+	// except for "events", "configmaps" and "secrets" so we'll gather those manually.
+	b := resource.NewBuilder(o.configFlags).
+		Unstructured().
+		ResourceTypeOrNameArgs(true, "all").
```

Comment:
try `.All`

Reply:
> try `.All`

oh, nm. that's cool.  does that work?

Reply:
it works, collects all resources we expect from a namespace + bc, dc, and routes. (We skip routes down below to collect and redact their info manually)